plugin/providers/runscope/data_source_runscope_integration.go

```diff
@@ -16,10 +16,32 @@ func dataSourceRunscopeIntegration() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"filter": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"values": {
+							Type:     schema.TypeList,
+							Required: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+					},
+				},
+			},
 			"type": {
```

Comment:
now that we have filters, do you think we should get rid of type as a top level schema attribute?

Reply:
looks great :) btw!

Reply:
I think using filter is more intuitive, but would not be backwards compatible so left `type` in to support both

Reply:
makes sense, thanks for the PR